README_template.md

```diff
@@ -0,0 +1,40 @@
+# Project Name
+One paragraph description of the project goes here.
+
+## Written in
+Programming Language
+Using the folowing libraries:
+* [pandas](http://pandas.pydata.org/)
+* [numpy](http://www.numpy.org/)
+* ...
+
+## Installation
+A step by step guide on how to get your project working on a new machine.
+Note: Write this with someone else, using their machine. It is easy to forget
+dependencies.
+
+End this section with a simple example of how to get your project to do
+something useful.
+
+## Tests
```

Comment:
Can we add something on QA activities - i.e. a description of the QA activities which were conducted.  And a link to the AQA checklist `.md` file which they should have included (and which they can get from here: https://moj-analytical-services.github.io/our-coding-standards/web/)